contents.json

```diff
@@ -5145,6 +5145,11 @@
       "category": "audio",
       "description": "An overlay that gets your user’s voice permission and input as text in a customizable UI.",
       "homepage": "https://github.com/algolia/voice-overlay-ios"
+    }, {
+      "title": "LightRoute",
+      "category": "app-routing",
+      "description": "LightRoute is easy transition for your app.",
```

Comment:
Hello @serhii-londar,

I'm not sure, but on the framework page, I found the following description: "LightRoute is easy transition **between VIPER modules**, who implemented on pure Swift. We can transition between your modules very easy from couple lines of codes.". So, this framework is more about transitioning between VIPER modules. 

What do you think about changing the description to something like: **"Routing between VIPER modules."**?

Reply:
Agree with you. Thank for the suggestion.